src/main/java/taskalyn/Parser.java

```diff
@@ -0,0 +1,153 @@
+package taskalyn;
+
+public class Parser {
+    private TaskManager taskManager;
+    private Ui ui;
+
+    public Parser(Ui ui, TaskManager taskManager) {
+        this.ui = ui;
+        this.taskManager = taskManager;
+    }
+    public boolean parse(TaskManager taskManager) {
```

Comment:
I like how clean this code segment is in this method!

Reply:
Thanks!